lib/ui/avatar.tsx

```diff
@@ -17,6 +18,7 @@ export const sizes = [
 const avatarVariants = cva("relative flex shrink-0 overflow-hidden", {
   variants: {
     size: {
+      xxsmall: "h-6 w-6 rounded-xl text-base text-white",
```

Comment:
Maybe we should just shrink the `xsmall` size? We don't want that many variations around, do we?

Reply:
Makes sense to me.